examples/with-typescript/package.json

```
@@ -10,13 +10,13 @@
   "dependencies": {
     "isomorphic-unfetch": "3.0.0",
     "next": "latest",
-    "react": "^16.10.1",
-    "react-dom": "^16.10.1"
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
   },
   "devDependencies": {
-    "@types/node": "^12.7.8",
-    "@types/react": "^16.9.3",
-    "@types/react-dom": "^16.9.1",
+    "@types/node": "^12.12.21",
+    "@types/react": "^16.9.16",
+    "@types/react-dom": "^16.9.4",
     "typescript": "3.6.3"
```

```suggestion
     "typescript": "3.7.3"
```
It would probably also be good to bump TypeScript to the latest version